### DISCUSSION TITLE
Group 2 Project Feedback: Payment System Options for Daya Users

### DISCUSSION THREAD

**Discussion by bbdaniels** [gui2de/ppol768-spring23#154] [Group Projects Feedback]
Group 2 -- This is a very creative project, but can you please add a short descriptive title so I know what it is about when I start to read it? Some feedback here:

I am not sure I understand the "problem". Why do women need to receive international payments? What share of women are employed? Is something stopping women from becoming employed (extensive margin), or are we looking at profits for women who are self-employed (intensive margins)? What is the current structure of the commercial transactions we are interested in? Who are the buyers, intermediaries, sellers, and producers? Fleshing this out will help define the structure of the study quite a bit more clearly, such as putting together the eligible population and the treatment and control groups.

That said, I like the structure of the project. I am going to echo my comments to Group 1 here, however, in that I worry that the idea of "market failure" isn't here in the right way, which is OK because it is a hard concept that I would like to see developed more in the next drafts. Specifically, the intervention is a bit unclear. Is the idea to make mobile money available to some subset of women? If so, why is that not already available / why is takeup insufficient? There must be some idea of market failure as to why this has not already happened -- given that you believe that such tools would improve income (or employment) for these womenn (and that mobile phone penetration is already high).

The best place to flesh this out is in the design. You say "women will employ mobile money services" -- but what does that mean? Is it the case that nobody currently is? How will you be making that more available? What are the volumes/costs of these transactions? Effectively, you need to answer the question of "why do you know better than what these women are currently chasing to do" -- the only way to justify that answer is "there is something preventing them from doing what is best", and _that_ is market failure.

Finally, you should of course flesh out the data. This needs to be much more substantial, particularly in describing exactly which people you are interested in. Are you interested in only people who are already entrepreneurs? If so, you should revise your introduction a bit (since you are not any more talking about increasing employment, but only increasing productivity / profits for those already employed). If not, should you think about a broader unit of analysis, such as providing mobile money intervention at some settlement level, and looking for impacts also on employment/entrepreneurship choices? You might also think carefully 

Again -- I like the project and think the concept is a great foundation to build out these questions. Looking forward to it!

**Comment by BeaLeydier** [comment]
Hi team

Great project! As somoene who started at gui2de working on our mobile money projects in Kenya, I was happy to find out your project was on this very topic! Building on Ben's comments above, which I absolutely second, I would add a couple specific things

1. In your motivation, it's not very clear to me how economic activity in particular and trade in general is related to banking usage in general and participation in global trade in particular. For example, this sentence :

> For which the EU Free trade agreement ([Grumiller et al., 2018](https://www.econstor.eu/bitstream/10419/268173/1/RR08.pdf)) provides an opportunity to access markets for rural entrepreneurs. However, there is a mismatch between large established exporters and smaller artisans that lack knowledge of how to navigate such a complicated banking system.

is confusing to me, because it sounds like you are saying the EU free trade agreement is a banking system? Which I don't think is what you mean, but distringuishing and fleshing out the relationships between economic activity, banking, trade, etc, will help you define your intervention and hypothesis more clearly.

2. In your intervention, I would think about what is the value you are bringing these women, why they haven't taken it up already, and how are you going to ensure that (1) they take up what you offer and (2) people who aren't offered it don't take it up on their own through other means than your intervention. Being more specific about the context will help to narrow it down, and that will respond to some of Ben's questions as well!

Cheers,
Beatrice

**Comment by lgf94** [comment]
Hello, apologies for my late reply, I did not know the feedback was going to be part of the discussion threads! 
I think we were not clear on why we came up with this. Last year Michele and I worked on a project, [Daya](https://drive.google.com/file/d/1hnvJjHcQVoV3LJ2Vnt7oSDs1CmM4fl3G/view?usp=sharing), where we created a social enterprise to support artisan women in Tunisia reach global markets. In very broad terms, it was a B2B marketplace like [poweredbypeople](https://www.poweredbypeople.com/) with a support and learning platform for the artisans. Here is where the EU free trade agreement comes into play, as the EU becomes the obvious market choice to start the venture. The biggest challenge we found to make this possible, and to truly empower the female artisans, was to define how they can receive international payments as Tunisia is not "connected" to the international financial system. In terms of the market failure, we gathered from our interviews to IFC and AidtoArtisans teams working in Tunisia, women artisan would either sell to male store owners in the touristic bazaar market mainly in Tunis, or to male-owned large established exporters who have the capacity to navigate the complex system. Daya would allow female artisans to sell their products directly, and to receive a direct payment without intermediates. Working with the people in Tunisia (and a payment system expert in Iran since they have a similar closed banking system) we defined that the best two payment options to allow this direct payment were (1) the post offices (Tunisia has the largest ["postal bank network"](https://www.worldbank.org/en/country/tunisia/publication/financial-inclusion-in-tunisia) in the world, with almost half the population owning a postal account) (2) the mobile payment system (smartphone penetration is over 92%, over 85% in rural areas).  

The intervention would be within the framework of Daya (now I see it cannot be outside of it as Daya solves the market failure), to explore which payment solution truly brings benefits to the female artisans using Daya. These findings could then have wider policy implications in the study of female entrepreneurship, empowerment and financial inclusion. 

I hope this now makes more sense, so my question now is, can we still do this project within this framework?

I know there are no more office hours until the deadline, I would have gone to say all of this in person, but I did not realise this would be on the discussion thread, since I am used to receiving the feedback privately. I only realised once I sat to go through this week's assignment and it was asking to adapt to the feedback given. Sorry I should have asked where to find it and if it was  ready.

thank you!

**Reply by bbdaniels** [reply]
Hi @lgf94 -- although we do not have _drop-in hours_, you can still book individual office hours with us over break. The only change for me is that I will not be on campus -- I will be downtown Tuesday instead as well as Thursday. Just let one of us know if you would like to schedule something with your group.

**Comment by bbdaniels** [comment]
# Concept Note Review

As we discussed I think this is much improved. It is now clear that this is a design interested in "horse-racing" two possibilities for mobile money handling within a program context. The target sample is clear and the comparison is also clear.  The main problem is that you have hidden the RCT design in a very small line, and if you are doing an RCT with recruitment into a larger program that has to be much more detailed. Are the people you are targeting already part of Daya? This is not really clear to me (or if it is contingent on the roll out of that program).

If you build out the actual program and treatment, you will have a very solid proposal from this. What is the total target group? Why do Daya users need to be "referred" to the experiment (instead of A/B testing through Daya itself). Why do you need consent for this, if it is a secondary-data business-case design? Why are you limiting the sample to "between 900 and 1200 people"? Why are you "weighting for population characteristics" (which characteristics)? Are you more interested in "new users" or "existing users"? What functionality is available to people who are not part of the experimental sample?

I very much like the approach of data collection through the app as well. Again, if you can clarify who you are able to reach, this will be great. The only issue here is that you have a broad-ranging and very qualitative-seeming survey that will not necessarily provide a ton of information that is useful. Why not narrow this down to a five-minute quantitative survey and administer it at (quasi-) random times through the app (referring to, say, the last week only)? This will help look at trends and uptake over time, such as speed of switching to payment through the app (vs outside the app, I assume).

For timeline -- you don't really need staff? Just run this over the course of a year fully through the app and then use the linked administrative data for app utilization (does this have sales and stuff?) to put together your analytical dataset. I think that will work very nicely. Great job!
